### ISSUE TITLE
Hyphenation on Google Chrome on non-macOS platforms

### ISSUE THREAD

**Issue by kwkbtr** [vivliostyle/vivliostyle.js#569]
Use some polyfill?

**Comment by MurakamiShinyu** [comment]
Chrome supports hyphenation on Windows and Android since version 88. See  https://caniuse.com/css-hyphens and https://www.chromestatus.com/feature/5672891947417600.

A test sample: https://vivliostyle.github.io/vivliostyle_doc/samples/wood/index.html
Test with Vivliostyle Viewer:  https://vivliostyle.org/viewer/#src=https://vivliostyle.github.io/vivliostyle_doc/samples/wood/index.html

Screen shot on Windows:
![Screen Shot 2021-03-01 at 11 02 28](https://user-images.githubusercontent.com/3324737/109449974-23879e80-7a8d-11eb-9c64-a224d38f9a6e.png)

As shown in this screen shot, hyphenation is enabled in the Chrome version 88.0.4324.190 on Windows.

So this issue should have already been resolved... however, I found hyphenation is still not enabled with Vivliostyle CLI 3.1.1 that uses Chromium 90.0.4403.0 via Puppeteer 7.0.1.
This seems to be a puppeteer problem. I checked puppeteer issues and found https://github.com/puppeteer/puppeteer/issues/6840 (Puppeteer does not honor hyphens:auto).

I tested also the latest Puppeteer 8.0.0 with Chromium 90.0.4427.0 and confirmed the problem still exists.

**Comment by MurakamiShinyu** [comment]
> This seems to be a puppeteer problem. …

No, this would be a Chromium headless mode problem. I reported it to Chromium bugs:
- [Issue 1261480: Automatic hyphenation not working in headless mode](https://bugs.chromium.org/p/chromium/issues/detail?id=1261480)

**Comment by MurakamiShinyu** [comment]
- https://github.com/vivliostyle/vivliostyle-cli/issues/551#issuecomment-2585285083

  > Switching Playwright to Puppeteer might be a solution of this problem.
  > 
  > Puppeteer uses Chrome for Testing which includes hyphen-data. I've tested Puppeteer on Windows and confirmed that hyphenation is working.

- https://github.com/vivliostyle/vivliostyle-cli/issues/553

**Comment by MurakamiShinyu** [comment]
This isssue is resolved in https://github.com/vivliostyle/vivliostyle-cli/pull/602 and [Vivliostyle CLI v10](https://github.com/vivliostyle/vivliostyle-cli/releases).
